src/reconstruction/refinement/addMultipleMetabolites.m

```diff
@@ -0,0 +1,78 @@
+function newmodel = addMultipleMetabolites(model,metIDs,varargin)
+% Simultaneously add multiple metabolites. Allows the use of the field
+% names of the model struct to specify additional properties of the
+% metabolites added.
+% USAGE:
+%
+%    model = addMultipleMetabolites(model,metIDs,varargin)
+%
+% INPUTS:
+%    model:     The model to add the Metabolite batch to.
+%    metIDs:    The metabolite IDs to add. No duplicate IDs may be provided
+%               and no ID may be present in the supplied model.
+%
+% OPTIONAL INPUTS:
+%    varargin:  fieldName, Value pairs. 
```

Comment:
What are the possible options here?

Reply:
all valid field names associated with mets specified in the Field Spec.